File: to-fhir/src/main/java/io/github/dizuker/tofhir/FhirSystems.java
package io.github.dizuker.tofhir;

/** Canonical FHIR coding system URIs used across to-FHIR®. */
public final class FhirSystems {
  /** The UCUM system. */
  public static final String UCUM = "http://unitsofmeasure.org";

  /** The LOINC system. */
  public static final String LOINC = "http://loinc.org";

  /** The SNOMED CT system. */
  public static final String SNOMED = "http://snomed.info/sct";

  /** The ATC system. */
  public static final String ATC = "http://fhir.de/CodeSystem/bfarm/atc";

  /** The OPS system. */
  public static final String OPS = "http://fhir.de/CodeSystem/bfarm/ops";

  /** The ICD-10-GM system. */
  public static final String ICD10GM = "http://fhir.de/CodeSystem/bfarm/icd-10-gm";

  /** The PZN system. */
  public static final String PZN = "http://fhir.de/CodeSystem/ifa/pzn";

  private FhirSystems() {
    // Utility class, prevent instantiation
  }
}

File: to-fhir-cs/ToFhir/FhirSystems.cs
namespace ToFhir;

/// <summary>Canonical FHIR coding system URIs used across to-FHIR®.</summary>
public static class FhirSystems
{
    /// <summary>The UCUM system.</summary>
    public const string Ucum = "http://unitsofmeasure.org";

    /// <summary>The LOINC system.</summary>
    public const string Loinc = "http://loinc.org";

    /// <summary>The SNOMED CT system.</summary>
    public const string Snomed = "http://snomed.info/sct";

    /// <summary>The ATC system.</summary>
    public const string Atc = "http://fhir.de/CodeSystem/bfarm/atc";

    /// <summary>The OPS system.</summary>
    public const string Ops = "http://fhir.de/CodeSystem/bfarm/ops";

    /// <summary>The ICD-10-GM system.</summary>
    public const string Icd10Gm = "http://fhir.de/CodeSystem/bfarm/icd-10-gm";

    /// <summary>The PZN system.</summary>
    public const string Pzn = "http://fhir.de/CodeSystem/ifa/pzn";
}
